crates/relayer-types/src/applications/ics27_ica/msgs/register.rs

```diff
@@ -54,16 +41,14 @@ impl TryFrom<RawMsgRegisterInterchainAccount> for MsgRegisterInterchainAccount {
     type Error = Error;
 
     fn try_from(value: RawMsgRegisterInterchainAccount) -> Result<Self, Self::Error> {
-        let chan_ordering = Ordering::from_i32(value.order).map_err(Error::ics04_channel)?;
-
         Ok(MsgRegisterInterchainAccount {
             owner: value.owner.parse().map_err(Error::owner)?,
             connection_id: value
                 .connection_id
                 .parse()
                 .map_err(Error::invalid_connection_identifier)?,
             version: value.version.into(),
-            order: chan_ordering,
+            order: Ordering::from_i32(value.order).unwrap_or_else(|_| Ordering::Ordered),
```

Comment:
I think it would be better to map and return the error if the conversion fails to avoid missing potential issues